lima.sh

```diff
@@ -5,10 +5,14 @@ case "$(uname)" in
   Darwin) display=cocoa;;
   Linux) display=gtk;;
 esac
+case "${ARCH}" in
+  x86_64) bios=true;;
+  *) bios=false;;
+esac
```

Comment:
I think we can drop setting `legacyBIOS: true` for all platforms now; it is no longer needed and has already been removed from the `examples/alpine.yaml` template in the lima.git repo.